packages/toolkit/src/view/pipeline/view-pipeline/ViewPipeline.tsx

```diff
@@ -81,8 +81,8 @@ export const ViewPipeline = () => {
         }}
         currentVersion={currentVersion}
       />
-      <div className="mx-auto flex flex-1 flex-row px-8">
-        <div className="flex h-full w-[718px] flex-col gap-y-6 py-10 pr-10">
+      <div className="flex flex-1 flex-row px-8">
+        <div className="w-3/5 gap-y-6 py-10 pr-10">
```

Comment:
@iamnamananand996 I think you misunderstand the demanded change, please take a look at how Hugging face maintain a fixed width on their page. Something like this 

https://huggingface.co/meta-llama/Meta-Llama-3-8B

Reply:
hugging face is using bellow.
```
md:col-span-7 
md:col-span-5
```

it is same as `w-7/12` and `w-5/12`, updated it now